lib/firefox_profile.js

```diff
@@ -152,7 +151,7 @@ function FirefoxProfile(options) {
     this.profileDir = opts.destinationDirectory || this._createTempFolder();
   } else {
     // create copy
-    var tmpDir = opts.destinationDirectory || this._createTempFolder('-copy');
+    var tmpDir = opts.destinationDirectory || this._createTempFolder('copy-');
```

Comment:
I had to change this to be a prefix instead of suffix due to how the API works.

This path is meant to be temporary and hidden from the user, so I'm guessing it was implemented only to improve debugging but isn't fundamental.

Reply:
The idea here is to not alter an existing profile directory (if provided in `options.profileDir`).